.github/workflows/alpha-release.yml

```diff
@@ -100,10 +105,10 @@ jobs:
     name: Release Github
     if: ${{ needs.changelog.outputs.skipped != 'true' }}
     runs-on: ubuntu-latest
-    needs: [build, changelog]
+    needs: [ build, changelog ]
     steps:
       - name: Publish Release
-        uses: eregon/publish-release@v1
+        uses:  eregon/publish-release@v1.0.6
```

Comment:
Actually, this pins this release as well, since `@1` will allow for any minor or patch release after that. Is this what you want? (For lesser-known mods, it could be a point, but I would like to know that it was intentional).

If you want to keep it, please revert the extra space.